.environment-scripts/install_local_packages.sh

```diff
@@ -19,6 +19,7 @@ pip install -c constraints.txt \
   -e external/report \
   -e external/synth \
   -e external/vcm \
+  -e external/mappm \
```

Comment:
If and when I commit m1 environment files, we can put this install inside bash logic.